stix_shifter_modules/sentinelone/README.md

```diff
@@ -0,0 +1,679 @@
+# SentinelOne Connector
+
+**Table of Contents**
+
+- [SentinelOne API Endpoints](#SentinelOne-api-endpoints)
+- [Pattern expression with STIX attributes - Single Observation](#single-observation)
+- [Pattern expression with STIX attributes - Multiple Observation](#multiple-observation)
+- [Pattern expression with STIX attributes - Execute Query](#stix-execute-query)
+- [Limitations](#limitations)
+- [References](#references)
+
+### SentinelOne API Endpoints
+
+   |Connector Method|SentinelOne API Endpoint| Method
+   | ----           |   ------              | -----|
+   |Query Endpoint  |https://<{fqdn}>/web/api/v2.1/dv/init-query/|POST
+   |Result Endpoint|https://<{fqdn}>/web/api/v2.1/dv/events/|GET
+   
+### Format for calling stix-shifter from the command line
+```
+python main.py `<translate or transmit>` `<translator_module>` `<query or result>` `<STIX identity object>` `<data>`
+```
+### Pattern expression with STIX attributes
+
+### Single Observation
+
+#### STIX Translate query
+```shell
+translate sentinelone query '{}' "[ipv4-addr:value = '1.1.0.0'] START t'2022-03-01T11:00:00.000Z' STOP t'2023-03-08T11:00:00.003Z'"
+```
+#### STIX Translate query - output
+```json
+{
+    "queries": [
+        "{\"query\": \"(srcIp = \\\"1.1.0.0\\\" OR dstIp = \\\"1.1.0.0\\\") AND EventTime  BETWEEN \\\"2022-03-01T11:00:00.000Z\\\" AND \\\"2023-03-08T11:00:00.003Z\\\"\", \"fromDate\": \"2022-03-01T11:00:00.000Z\", \"toDate\": \"2023-03-08T11:00:00.003Z\", \"limit\": 10000}"
+    ]
+}
+```
+#### STIX Transmit ping 
+
+```shell
+transmit
+sentinelone
+"{\"host\":\"xx.xx.xx\"}"
+"{\"auth\":{\"apitoken\": \"xxxxx\"}}"
+ping
+```
+
+#### STIX Transmit ping - output
+```json
+{
+    "success": true,
+    "code": 200
+}
+```
+#### STIX Transmit query 
+
+```shell
+transmit
+sentinelone
+"{\"host\":\"xx.xx.xx\"}"
+"{\"auth\":{\"apitoken\": \"xxxxx\"}}"
+query
+"{\"query\": \"(srcIp = \\"1.1.0.0\\" OR dstIp = \\"1.1.0.0\\") AND EventTime  BETWEEN \\"2022-03-01T11:00:00.000Z\\" AND \\"2023-03-08T11:00:00.003Z\\"\", \"fromDate\": \"2022-03-01T11:00:00.000Z\", \"toDate\": \"2023-03-08T11:00:00.003Z\"}"
+```
+
+#### STIX Transmit query - output
+```json
+{
+    "success": true,
+    "search_id": "xxxxxx"
+}
+```
+#### STIX Transmit status 
+
+```shell
+transmit
+sentinelone
+"{\"host\":\"xx.xx.xx\"}"
+"{\"auth\":{\"apitoken\": \"xxxxx\"}}"
+status
+"xxxxxx"
+```
+
+#### STIX Transmit status - output
+```json
+{
+    "success": true,
+    "status": "FINISHED",
```

Comment:
I would expect this to come back as "COMPLETED" not "FINISHED"

Reply:
Changed the text in README.md